OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -135,6 +144,12 @@ int main()
         Model modelWithContacts("visualize_contacts.osim");
         populate_contactModelPrimitives(standard);
         testVisModelAgainstStandard(modelWithContacts, standard);
+        
+        // Model with WrapObjects
+        Model modelWithWrap("test_wrapAllVis.osim");
+        modelWithWrap.updDisplayHints().set_show_frames(false);
```

Comment:
If show_frames == false, why are there so many DecorativeFrames in `populate_wrapModelPrimitives`?